copis/core.py

```diff
@@ -251,7 +265,7 @@ def _sender(self) -> None:
             while self._machine and self.imaging and not self._clear:
                 time.sleep(0.001)
 
-    def start_imaging(self, startindex=0) -> bool:
+    def start_imaging(self) -> bool:
```

Comment:
Leave in startindex (need some way of imaging only a certain subset of actions). Or we could think of a better way to handle this?

Reply:
We can always add it back when we are actually using it in the method.